docs/core/tools/dotnet-install-script.md

```diff
@@ -100,8 +100,8 @@ The install scripts do not update the registry on Windows. They just download th
 
   Specifies the source channel for the installation. The possible values are:
 
-  - `Current` - Most current release.
-  - `LTS` - Long-Term Support channel (most current supported release).
+  - `STS` - Standard Term Support SDK
+  - `LTS` - The most current Long Term Support release
```

Comment:
lets replace the word "current" with recent.

Reply:
This change too should be made in the scripts themselves as well.